provisioning/plugins/corim-psa-decoder/test-harness/build-test-vectors.sh

```diff
@@ -34,3 +35,25 @@ do
 	echo '`' >> ${TV_DOT_GO}
 	gofmt -w ${TV_DOT_GO}
 done
+
+CORIM_TEMPLATE=corimCca.json
+CORIM_TEMPLATE1=corimCcaNoProfile.json
+
+COMID_TEMPLATES1=
+COMID_TEMPLATES1="${COMID_TEMPLATES1} ComidCcaRefValOne"
+COMID_TEMPLATES1="${COMID_TEMPLATES1} ComidCcaRefValFour"
+
+for t in ${COMID_TEMPLATES1}
```

Comment:
Need to optimize this by extracting out in a function!

Reply:
yes, please 👍